### DISCUSSION TITLE
Support custom docker arguments

### DISCUSSION THREAD

**Discussion by RayBB** [coollabsio/coolify#1679] [Feature Requests]
Add option for custom docker arguments

For example:
`docker run --cap-add SYS_ADMIN --device /dev/fuse --security-opt apparmor:unconfined`
 
This is [necessary](https://stackoverflow.com/questions/48402218/fuse-inside-docker) for me to run rclone mount in a container with FUSE.

I asked about this in Discord [here](https://discord.com/channels/459365938081431553/1199740440241516635/1199740440241516635) and eventually found [this message](https://discord.com/channels/459365938081431553/1192458904249057291/1196765165786517515) where @andrasbacsai said it would be supported eventually.

Thought I'd post this request on GitHub so others can easily find it and upvote to show they want it.

**Comment by andrasbacsai** [comment]
I will add initial support for a few options in v202. More info [here](https://coolify.io/docs/custom-docker-options).

**Reply by svallory** [reply]
It looks like I can only set options when I'm building from code. Can we have that for docker images too?

**Comment by jcunix** [comment]
I believe this is what I am running into as well.

**Reply by RayBB** [reply]
What problem are you having?
